src/compiler/scala/tools/nsc/ast/DocComments.scala

```diff
@@ -454,17 +454,17 @@ trait DocComments { self: Global =>
       }
 
       def getSite(name: Name): Type = {
-        def findIn(sites: List[Symbol]): Type = sites match {
-          case site1 :: sites1 => select(site1.thisType, name, findIn(sites1))
```

Comment:
why did you inline `select`?

Reply:
I didn't like the recursion through `orElse`; I think I intended to eliminate `select` but it is used below. I neglected to return to this spot.

Reply:
oh I returned in the second commit but did not uninline select.